tests/TestHelpers/GraphHelper.php

```diff
@@ -2040,4 +2040,63 @@ public static function getDrivePermissionsList(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $spaceId
+	 * @param array $shareeIds
+	 * @param array $shareTypes
+	 * @param string|null $permissionsRole
+	 * @param string|null $permissionsAction
+	 * @param string|null $expireDate
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function sendSharingInvitationForDrive(
```

Comment:
i see a lots of code repetition for this function with the already existing function `sendSharingInvitation`. is it possible to work around using the existing function or its not possible? Would be good if we use the existing function with some workaround?

Reply:
> Would be good if we use the existing function with some workaround?

hmm, but let's not do workarounds for different endpoints
maybe some codes can be moved to methods but that can be in next PRs